### DISCUSSION TITLE
Connecting identity_mapping_v3 to identity provider

### DISCUSSION THREAD

**Discussion by tasansga** [opentelekomcloud/terraform-provider-opentelekomcloud#1053] [Q&A]
I'm currently working on SSO with SAML to our OTC platform. The test setup, clicked together in the OTC console, works fine.

But I'm unable to replicate it with terraform. Specifically how can I map a https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/resources/identity_mapping_v3 to a specific identity provider? When I create mappings in terraform they are created but not connect to one of the providers, they appear nowhere. How does the mapping_id actually relate to the identity provider?

Thanks!

**Comment by outcatcher** [comment]
From the API point of view, federation seems to be a copy of OpenStack federation, with mappings not having reference to the provider. At the same time, console mappings are created for the exact provider. I've raised a question internally: https://jira.tsi-dev.otc-service.com/browse/OTCSEC-1905. Will update you when got the answer.
